### ISSUE TITLE
Extra folders

### ISSUE THREAD

**Issue by bil-elmoussaoui** [mank319/elementaryPlus#86]
Hello, 
i would like to ask if any good designer wants to add one of those folders : 
- [ ] Dropbox
- [ ] Megasync 
- [ ] Movies/Series
...
There's some folders on this deviantart page : http://fabianinostroza.deviantart.com/art/Elementary-Extra-Folders-387250061
The designer have given you the permission to use those icons "ok (y) ... but this folders are in png ... not svg,but works ;)"
So you might use this to get some inspirations for the folders ;) 
Thank you!!

**Comment by RafaelCz** [comment]
IMHO: as long as pantheon doesn't support custom folders, it's a waste of time.

**Comment by quassy** [comment]
Is there a bug on Launchpad we can keep an eye on? I can't find one [over there](https://bugs.launchpad.net/elementary).

**Comment by bil-elmoussaoui** [comment]
I don't think so. But if you use Nautilus/Nemo...The icons works fine. If you create the issue share it here to keep an eye on it too

**Comment by bil-elmoussaoui** [comment]
I add a bounty on this ;)

**Comment by quassy** [comment]
Maybe you can convince someone to convert [these icons on DeviantArt](http://fabianinostroza.deviantart.com/art/Elementary-Extra-Folders-387250061) to SVG.

**Comment by bil-elmoussaoui** [comment]
That's exactly what i want, if anyone wants to work on this and maybe adding more folders too ;)

**Comment by bil-elmoussaoui** [comment]
I reported this to pantheon-files, please mark it as "it's affects me" so this can be fixed asap
https://bugs.launchpad.net/pantheon-files/+bug/1462824

**Comment by elmat0** [comment]
I have some free time over the next few days so I'm happy to convert those icons over to SVG if no one else is working on this. Someone else will have to actually merge them into the theme etc, or at least walk me through that.

I don't have nautilus installed so I can't test them until (I live in hope) pantheon-files has places support.

**Comment by bil-elmoussaoui** [comment]
@elmat0 there's no one working on that at the moment , feel free to start working on this ;)

**Comment by elmat0** [comment]
Ok. I didn't get through all of your request but I'll be away fishing for a week and figured you might as well look through what I've done. Only 128px versions included for now. Hack them apart if you like. Be critical. (I'm not easily offended ;p) Post up some suggestions etc. 

[http://elma.to/Extra_Folders-128-v1.zip](Download)

**Comment by elmat0** [comment]
Just realised my SSL key has expired... :S

**Comment by bil-elmoussaoui** [comment]
They're amazing!!! If you can add just two folders a viber & telegram one :P 
Screenshots for the current folders
![capture du 2015-07-09 16 51 36](https://cloud.githubusercontent.com/assets/7660997/8598398/d4b3f394-265a-11e5-972b-6b799a1ff732.png)
![capture du 2015-07-09 16 51 46](https://cloud.githubusercontent.com/assets/7660997/8598414/e4bae266-265a-11e5-9172-1011e031fa0e.png)
![capture du 2015-07-09 16 51 58](https://cloud.githubusercontent.com/assets/7660997/8598418/e6ee70b6-265a-11e5-9721-a58d187106c4.png)
Just one thing, the samllest size (16x16) must be icons like this
![capture du 2015-07-09 16 53 35](https://cloud.githubusercontent.com/assets/7660997/8598447/0c5fe8d4-265b-11e5-9a5a-7017a6113c8d.png)
instead of folder icons

**Comment by quassy** [comment]
Pretty cool that someone is working on this! I can help you with creating a Pull Request or if you prefer create a new branch with your icons (including proper attribution of course) myself. 

Do you know if there are any conventions for naming these icons? The only similar icon we have is currently named `insync-folder.svg` (so the other way round).

**Comment by RafaelCz** [comment]
There is no name convention for folder icons yet, since elementary OS doesn't support it...
I still think creating them is wasted time and effort, at least until there is no custom folder support.
@elmat0 please don't get me wrong, I like really them :)

**Comment by bil-elmoussaoui** [comment]
@quassy i do think that folders must be named like that, this a list of icons in ``` /usr/share/icons/elementary/places/32```
```
bookmark-missing.svg     folder-templates.svg
distributor-logo.svg     folder-videos.svg
folder-documents.svg     gtk-network.svg
folder-download.svg      inode-directory.svg
folder-drag-accept.svg   library-music.svg
folder-music.svg         network-server.svg
folder-open.svg          network-workgroup.svg
folder-pictures.svg      user-bookmarks.svg
folder-publicshare.svg   user-desktop.svg
folder-recent.svg        user-home.svg
folder-remote.svg        user-trash-full.svg
folder-saved-search.svg  user-trash.svg
folder.svg
``` 
I guess that insync-folder must be renamed. But it's not that important since this is not fixed yet in Files

**Comment by quassy** [comment]
> I still think creating them is wasted time and effort, at least until there is no custom folder support.
@elmat0 please don't get me wrong, I like really them :)

Disagree, also people who don't use Pantheon Files (either because they have installed Nautilius or even another OS) should have the chance to taste the maximum greatness of elementary + elementaryPlus icons ;)

**Comment by elmat0** [comment]
@bil-elmoussaoui Aaah I didn't realise the 16px versions were different. A job for another day I guess. ;p

@quassy That would be great if you could upload them somewhere. I spent about 30 mins trying to figure out git but ran out of time. We should probably attribute the deviantart guy also. One thing... what are the legalities of using 3rd party logos? (twitter etc.)

@RafaelCz Build it and they will come? ;p In honesty I just wanted an excuse to learn Inkscape. Nice little program.

**Comment by elmat0** [comment]
Also.. I'm not really a fan of the `sites` or `virtualbox-alt` icons.

**Comment by quassy** [comment]
I'm not a fan of `folder-mega_alt2` ;). Regarding `folder-sites`: You could get rid of the cursor and just call it `folder-web` or similar. 

Some icons I'd like to use (if you are still motivated): 
* [ ] Archive (don't really know what icon to use here)
* [ ] Backup
* [ ] Bitbucket
* [ ] Fonts
* [ ] Paintings (maybe use a [palette](https://cdn3.iconfinder.com/data/icons/basic-mobile-part-2/512/painter-512.png) icon)
* [ ] Radio
* [ ] School
* [ ] University
* [ ] Work

> That would be great if you could upload them somewhere.

I have created a branch here with your icons.

> I guess that insync-folder must be renamed. But it's not that important since this is not fixed yet in Files

Will do.

**Comment by bbjansen** [comment]
Hello,

I been trying to find a copy of the extra folders everywhere. The original link to the DeviantArt does not exist anymore, and I don't seem them either in the branch. 

Thanks!

**Comment by elmat0** [comment]
The 128px versions I converted can be found here. https://github.com/elmat0/icons_etc/tree/master/places

I mocked up a few 16px ones though lost interest in elementary after a while. You are welcome to finish them and merge into elementaryPlus.
